app/js/controllers.js

```diff
@@ -0,0 +1,139 @@
+floripaApp.controller("GMapController", ['$scope', 'uiGmapGoogleMapApi', '$location', 'searchService', function ($scope, uiGmapGoogleMapApi, $location, searchService) {
+
+    $scope.goBack = function () {
+        $location.url('/');
+    };
+
+    $scope.search = function () {
+        searchService.streetName = $scope.streetName;
+        searchService.doSearch = true;
+        $location.url('/');
+    };
+
+    uiGmapGoogleMapApi.then(function (maps) {
```

Comment:
Oops, I don't. This is a point where it is guaranteed that the Google maps sdk has finished loading, it was there in case I needed it and I forgot to remove when cleaning the code.